lti_consumer/static/js/xblock_lti_consumer.js

```diff
@@ -133,8 +134,7 @@ function LtiConsumerXBlock(runtime, element) {
                     close: function(event, ui) {
                       $(this).remove();
                     }
-                  });
-
+                  }).prev().css('background', 'white').css('color', '#000').css('border-color', 'transparent');
```

Comment:
this gets rid of the "x-out" button in the top right corner. There is no easy way to do that.